src/form/Form.jsx

```diff
@@ -331,10 +331,10 @@ function Form({
     const fieldKeys = [];
     repeatedServarFields.forEach((field) => {
       const { servar } = field;
-      updatedValues[servar.key] = [
-        ...fieldValues[servar.key],
-        getDefaultFieldValue(field)
-      ];
+      updatedValues[servar.key] =
+        servar.type === 'file_upload'
+          ? fieldValues[servar.key]
```

Comment:
why aren't we adding an extra null or something to the array of files?

Reply:
it seemed weird to me to add a null value for the file, but I guess I should add a null to be consistent with the empty string that gets added for a repeated dropdown etc